src/components/utils/renderers/user-cell-renderer.tsx

```diff
@@ -4,33 +4,35 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import { Box } from '@mui/material';
+import { Avatar, Box, Theme, useTheme } from '@mui/material';
 import Tooltip from '@mui/material/Tooltip';
-import Chip from '@mui/material/Chip';
 
-const abbreviationFromUserName = (name: string | null) => {
-    if (name === null) {
-        return '';
+function getAvatarChildren(name: string) {
+    const splittedName = name.split(' ');
+    if (splittedName.length > 1) {
+        return `${splittedName[0][0]}${splittedName[1][0]}`;
```

Comment:
before it was first and last, now first and second ?

Reply:
fixed